corehq/apps/app_manager/templates/app_manager/simprints_enrolment_form.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>{{ form_name }}</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/{{ xmlns_uuid }}" uiVersion="1" version="1" name="{{ form_name }}">
+					<name />
+                    <simprintsEnrol />
+                    <simprintsId />
+                    <rightIndex />
+                    <rightThumb />
+                    <leftIndex />
+                    <leftThumb />
+				</data>
+			</instance>
+			<bind vellum:nodeset="#form/name" nodeset="/data/name" type="xsd:string" required="true()" />
+			<bind vellum:nodeset="#form/simprintsEnrol" nodeset="/data/simprintsEnrol" type="intent" required="true()" />
+			<bind vellum:nodeset="#form/simprintsId" nodeset="/data/simprintsId" />
```

Comment:
Is it intentional that these aren't required?

Reply:
no, i will make them required